src/Data/SkillStatMap.lua

```diff
@@ -554,6 +554,9 @@ return {
 	mod("PrimaryDuration", "BASE", nil, 0, 0, { type = "Multiplier", var = "CorpseConsumedRecently", limit = 4 }),
 	div = 1000,
 },
+["modifiers_to_skill_effect_duration_also_affect_soul_prevention_duration"] = {
```

Comment:
modifiers_to_buff_effect_duration_also_affect_soul_prevention_duration handling this one would be nice too, its on auras and stuff